### ISSUE TITLE
Log message to console: "Unable to parse the test name"

### ISSUE THREAD

**Issue by binarycow** [spekt/nunit.testlogger#90]
The following log messages are produced:

```
2021-11-22T23:55:39.7872697Z Xml Logger: Unable to parse the test name 'NetworkPrimitives.Tests.Common.CharExtensionTests.TestIsHex('"',False)' into a namespace type and method. Using Namespace='UnknownNamespace', Type='UnknownType' and Method='NetworkPrimitives.Tests.Common.CharExtensionTests.TestIsHex('"',False)'
2021-11-22T23:55:39.7882339Z Xml Logger: Unable to parse the test name 'NetworkPrimitives.Tests.Common.CharExtensionTests.TestIsHex('(',False)' into a namespace type and method. Using Namespace='UnknownNamespace', Type='UnknownType' and Method='NetworkPrimitives.Tests.Common.CharExtensionTests.TestIsHex('(',False)'
2021-11-22T23:55:39.7889575Z Xml Logger: Unable to parse the test name 'NetworkPrimitives.Tests.Common.CharExtensionTests.TestIsHex(')',False)' into a namespace type and method. Using Namespace='UnknownNamespace', Type='UnknownType' and Method='NetworkPrimitives.Tests.Common.CharExtensionTests.TestIsHex(')',False)'
```

I suspect it has to do with the usage of `(`, `)`, and `"`

- [The test method that is producing the log message](https://github.com/binarycow/NetworkPrimitives/blob/nunit-logger-bug/tests/NetworkPrimitives.Tests/Common/CharExtensionTests.cs#L236)
- [GitHub Actions logs](https://github.com/binarycow/NetworkPrimitives/runs/4293586350?check_suite_focus=true)
- The log messages are also produced on a local build (not using GitHub actions)
- The `dotnet test` command line arguments that are being used [are located here](https://github.com/binarycow/NetworkPrimitives/blob/0f33e5087a80f6a7208154963b6bd998285c3f08/.github/workflows/build-and-test.yml#L30)
- There are two test projects
  - The first `.csproj` file [is located here](https://github.com/binarycow/NetworkPrimitives/blob/nunit-logger-bug/tests/NetworkPrimitives.Tests/NetworkPrimitives.Tests.Core.csproj)
  - The second `.csproj` file [is located here](https://github.com/binarycow/NetworkPrimitives/blob/nunit-logger-bug/tests/NetworkPrimitives.Tests/NetworkPrimitives.Tests.Framework.csproj)
  - Both `.csproj` files are importing the `Directory.Build.props` [file located here](https://github.com/binarycow/NetworkPrimitives/blob/nunit-logger-bug/tests/Directory.Build.props).

**Comment by Siphonophora** [comment]
@binarycow Thanks for letting us know. I agree its probably the chars.

**Comment by binarycow** [comment]
@Siphonophora - Want me to leave my branch/PR (on my repo) open, so you can replicate/whatever?  Or, am I free to close?

**Comment by Siphonophora** [comment]
@binarycow Fine to close it. I have a fix on the core testlogger library for this, waiting on a code review.

**Comment by Siphonophora** [comment]
@binarycow If you have time to test it, we have an updated version of this logger here: https://ci.appveyor.com/project/spekt/nunit-testlogger/builds/41685956/artifacts

**Comment by becha2** [comment]
Hello, I came across this as I was looking to upgrade this package (related to: https://github.com/spekt/testlogger/issues/28).

I checked out the updated version there and one difference I could spot right away are tests that have apostrophes in the test name now become unable to be parsed:

```
[TestCase("example1", TestName = "Customer's name")]
public void MyTest(
    string a
    ) { ... }
```

**Comment by codito** [comment]
Potential fix in https://github.com/spekt/nunit.testlogger/pull/93

**Comment by codito** [comment]
Fix is released in 3.0.127.
